Bottom: 3060602d61af2ff12679d5e22aaeafb388a988e4
Top:    05935d27109b99372c41cc22ba585d5b1277866e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-15 16:28:53 +0100

Hack around Circuit CAM / EDG brokeness reading drill files







---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index e95186b..54c010f 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -462,10 +462,17 @@ gerber_set_layer (const char *name, int group)
 	      int ap = findApertureCode (pending_drills[i].diam, ROUND);
 	      fprintf (f, "T%02d\015\012", ap);
 	    }
+#ifdef EDG_ARE_BROKEN
+	  fprintf (f, "X%06ldY%06ld\015\012",
+		   100 * gerberX (PCB, pending_drills[i].x), 100 * gerberY (PCB,
+								pending_drills
+								[i].y));
+#else
 	  fprintf (f, "X%06ldY%06ld\015\012",
 		   gerberX (PCB, pending_drills[i].x), gerberY (PCB,
 								pending_drills
 								[i].y));
+#endif
 	}
       free (pending_drills);
       n_pending_drills = max_pending_drills = 0;
